WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -916,6 +894,7 @@ public void onPhotoPickerHidden() {
     public void onPhotoPickerMediaChosen(@NonNull final List<Uri> uriList) {
         mEditorPhotoPicker.hidePhotoPicker();
 
+        // TODO: It looks like we might be calling addMediaList twice
```

Comment:
I'm not sure I understand this comment correctly, but haven't you just missed the `return` statement after ```WPMediaUtils.advertiseImageOptimization(this, () -> mEditorMedia.addMediaList(uriList, false));```?

Reply:
I don't really remember what I was thinking with this. The `TODO` doesn't make sense 😕 Removed in 48f8d02d516a1ac3b0c3960910961cfe54bf4b9c.